cmd/proxy/main.go

```diff
@@ -38,15 +41,37 @@ func main() {
 		log.Fatal(err)
 	}
 
-	cert, key, err := conf.TLSCertFiles()
+	cert, key, mtlsCert, err := conf.TLSCertFiles()
```

Comment:
```suggestion
	cert, key, caCert, err := conf.TLSCertFiles()
```
It is not really an MTLS cert, but the CA cert.